lampo-bdk-wallet/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 log = "*"
 lampo-common = { path = "../lampo-common" }
-bdk = { version = "1.0.0-alpha.11", features = ["keys-bip39"] }
-bdk_chain = { version = "0.17" }
-bdk_esplora = { version = "0.16", features = ["blocking"]  }
-bdk_file_store = { version = "0.14" }
+bdk_wallet = { version = "1.0.0-beta.6", features = ["keys-bip39"] }
```

Comment:
Why did you choose this version? and not working with the alpha.11?

looks like that this version https://github.com/bitcoindevkit/bdk/blob/v1.0.0-beta.6/crates/bitcoind_rpc/Cargo.toml#L19 is not compatible with lampo

Reply:
my first attempt to go with `alpha.11` failed, due to `cargo update` error
```
$ cargo update
error: no matching package named `bdk_persist` found
location searched: registry `crates-io`
required by package `bdk v1.0.0-alpha.11`
    ... which satisfies dependency `bdk = "^1.0.0-alpha.11"` of package `lampo-core-wallet v0.1.0 (/home/mialso/projects/clones/lampo.rs/lampo-core-wallet)`
    ... which satisfies path dependency `lampo-core-wallet` of package `lampod-cli v0.1.0 (/home/mialso/projects/clones/lampo.rs/lampod-cli)`
```